src/cloud/docker/docker-config.md

```diff
@@ -17,9 +17,7 @@ The `docker:build` command overwrites the existing `docker-compose.yml` configur
 ## Prerequisites
 
 1. You must have the following software installed on your local workstation:
-   -  PHP version 7.1 or later
-      -  [php@7.1][php71]
-      -  [php@7.2][php72]
+   -  [PHP version 7.1 or later][php]
```

Comment:
so the individual links aren't useful?

Reply:
They go to brew installation instructions, and Don updated them in the master branch because they were broken. Also, the other links go to the product installation pages.